monkey/common/utils/secret_variable.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from pydantic.types import SecretStr
+
+from common.utils.i_secret_variable import ISecretVariable
+
+
+class SecretVariable(ISecretVariable):
```

Comment:
Theoretically this should be called something like `PydanticSecretVariable`, but It's unlikely that we'll need multiple instances of it, so I kept the name simple

Reply:
If it's unlikely that we'll need multiple implementations, why do we have an interface at all?
If we do decide to keep it, I say we rename this to `PydanticSecretVariable`.

Reply:
It somewhat achieves dependency inversion, which has all kinds of benefits and some drawbacks. The most trivial benefit is for example, that we would be able to create a test `MockSecretVariable` and pass it wherever `ISecretVariable` is expected. If there's no interface, we would have to mock the actual `SecretVariable`. You are right, we don't need the interface ATM, but I think it's better to have it and not need it, than to need it and not have it.

Reply:
Fair enough. I think we should rename it then.